Installer/build.ps1

```diff
@@ -26,6 +26,7 @@ if($null -eq $env:ZITI_EDGE_TUNNEL_BUILD) {
     }
     echo "Beginning to download ziti-edge-tunnel from ${zet_dl}"
     echo ""
+    $ProgressPreference = 'SilentlyContinue'
```

Comment:
Is this what avoids the prompt(s) to install things that are already installed?

Reply:
No. That's apparently an advanced installer bug. This prevents output to the shell with progress of download